### PR TITLE
Updating - tab-add-in-combined Sample

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

**Review by Prithvi-MSFT — APPROVED** [review]
changes look fine

File: .github/workflows/tab-add-in-combined-nodejs.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        with:
+          name: node-app
+
+      - name: 'Deploy to Azure Web App'
+        id: deploy-to-webapp
+        uses: azure/webapps-deploy@v2
+        with:
+          app-name: ${{ env.AZURE_WEBAPP_NAME }}
+          slot-name: 'production'
+          publish-profile: ${{ secrets.AzureAppService_PublishProfile_Tab_Add_In_Combined }}
```
Comment (ChetanSharma-msft) [inline .github/workflows/tab-add-in-combined-nodejs.yml:67]: We need to add this secret?

File: samples/tab-add-in-combined/nodejs/add-in/webpack.config.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ const CopyWebpackPlugin = require("copy-webpack-plugin");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
 
 const urlDev = "https://localhost:3000/";
-const urlProd = "https://www.blueyonderairlines.com/"; // CHANGE THIS TO YOUR PRODUCTION DEPLOYMENT LOCATION
+const urlProd = "https://tabdiscounts.z13.web.core.windows.net/"; // CHANGE THIS TO YOUR PRODUCTION DEPLOYMENT LOCATION
```
Comment (ChetanSharma-msft) [inline samples/tab-add-in-combined/nodejs/add-in/webpack.config.js:8]: Did you mention this in our Readme file?

File: samples/tab-add-in-combined/nodejs/appManifest/manifest.json [hunk 3]
```diff
@@ -61,7 +61,6 @@
         "messageTeamMembers"
     ],
     "validDomains": [
-        "${{ADDIN_DOMAIN}}",
         "${{TAB_DOMAIN}}"
```
Comment (ChetanSharma-msft) [inline samples/tab-add-in-combined/nodejs/appManifest/manifest.json:64]: This should be auto generated?

Comment (ChetanSharma-msft) [reply samples/tab-add-in-combined/nodejs/appManifest/manifest.json:64]: If yes, please make sure to use proper place holder.

File: samples/tab-add-in-combined/nodejs/assets/sample.json [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        "name": "Teams developer questions",
+        "url": "https://aka.ms/TeamsPlatformFeedback"
+      },
+      {
+        "name": "Teams development videos from Microsoft",
+        "url": "https://aka.ms/sample-ref-teams-vids-from-microsoft"
+      },
+      {
+        "name": "Teams development videos from the community",
+        "url": "https://aka.ms/sample-ref-teams-vids-from-community"
```
Comment (ChetanSharma-msft) [inline samples/tab-add-in-combined/nodejs/assets/sample.json:64]: This looks wrong.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comments

File: samples/tab-add-in-combined/nodejs/add-in/webpack.config.js (hunk 2 above)
Comment (Harikrishnan-MSFT) [reply samples/tab-add-in-combined/nodejs/add-in/webpack.config.js:8]: We are not using urlDev and urlProduction

Comment (Harikrishnan-MSFT) [reply samples/tab-add-in-combined/nodejs/add-in/webpack.config.js:8]: Fixed

File: samples/tab-add-in-combined/nodejs/appManifest/manifest.json (hunk 3 above)
Comment (Harikrishnan-MSFT) [reply samples/tab-add-in-combined/nodejs/appManifest/manifest.json:64]: ${{TAB_DOMAIN}} is auto generated. ${{ADDIN_DOMAIN}} we are not using.

File: samples/tab-add-in-combined/nodejs/assets/sample.json (hunk 4 above)
Comment (Harikrishnan-MSFT) [reply samples/tab-add-in-combined/nodejs/assets/sample.json:64]: Fixed

File: samples/tab-add-in-combined/nodejs/appManifest/manifest.json (hunk 3 above)
Comment (Harikrishnan-MSFT) [reply samples/tab-add-in-combined/nodejs/appManifest/manifest.json:64]: Fixed

File: .github/workflows/tab-add-in-combined-nodejs.yml (hunk 1 above)
Comment (Harikrishnan-MSFT) [reply .github/workflows/tab-add-in-combined-nodejs.yml:67]: Yes we need to add.

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving!

**Review by ChetanSharma-msft — APPROVED** [review]
Looks good

**Comment by Harikrishnan-MSFT** [comment]
**Build Pipeline -** 
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6324063978

image

**Pull request merged**